lib/diagnostic.js

```diff
@@ -28,7 +45,14 @@ const wordCount = function (/* */) {
 // 4. Write a function that takes a string and returns a dictionary with unique
 // words as keys and a count of each word as the value for that key
 const wordFrequencies = function (/* */) {
-  /* your response here */
+
+// Definitely the toughest diagnostic so far.
+// I don't think it's because I don't understand the concepts,
+// because when I follow along in class it seems to make sense to me.
+// But it will probably just take extra examples and extra practice
+// before all of this becomes second nature to me. I think if I had more time to
+// do this, I will def figure this out.
+
```

Comment:
No worries, it looks like you got the first two questions pretty well so you may just need some extra time. Please reach out if you're struggling with any concepts.